plinth/modules/bind/__init__.py

```diff
@@ -31,7 +31,9 @@
 
 version = 1
 
-title = _('Domain Name Server \n (BIND)')
+title = _('BIND')
```

Comment:
I didn't do it as it was changing a lot of files. As there are several modules that do not have description and were not requiring change.

But if it makes sense will do the refactoring and rename title to name.